LICENSE

```diff
@@ -1,6 +1,7 @@
 The MIT License (MIT)
 
 Copyright (c) 2015 Lukas Martinelli
+Copyright (c) 2024 MapLibre contributors
```

Comment:
Should this be only Maplibre or keep it with Lukas as it is now?

Reply:
Think just maplibre is fine in this case

Reply:
In all other repos we kept the old copyright holder and just added MapLibre. I think we should do the same here...

Reply:
we can never remove existing copyrights